packages/injected/src/helpers.ts

```diff
@@ -0,0 +1,23 @@
+import { ProviderRpcErrorCode, WalletModule } from "@sovryn/onboard-common";
+import { ProviderLabel } from "./types.js";
```

Comment:
remove please `.js` extension from import files, some build tools or devs with different IDE's may have issues with it in the future